### PR TITLE
Add basic walkthrough for the web interface

### PR DESCRIPTION
**Pull request by pcbailey**
Add a basic walkthrough for the web interface to include:

* VM/template CRUD
* NICs & disks management
* Console access

### CODE REVIEW

**Comment by jelkosz** [comment]
@fabiand can you please have a look / assign someone?

**Comment by fabiand** [comment]
@jelkosz @mareklibra can you sanity review this please

**Comment by fabiand** [comment]
> @fabiand can you please have a look / assign someone?

You look like a good candidate, not? .)

**Comment by jelkosz** [comment]
> You look like a good candidate, not? .)

:D was not aware that I count :)

**Comment by fabiand** [comment]
You are a pro in that area, thus who coul dbe better? :)

**Comment by jelkosz** [comment]
OK, so I have checked it. LGTM

**Comment by jelkosz** [comment]
@fabiand ok, added lgtm in two flavors :) is it good to go now?

**Comment by fabiand** [comment]
Let me find some nit picks to delay it by 2 weeks ...

File: web-interface/controlling-vms-modules/restart-vm-web.adoc [hunk 1]
```diff
@@ -0,0 +1,16 @@
+[[restart-vm-web]]
+== Restarting a Virtual Machine
+
+A running virtual machine can be xref:vm-actions-web[restarted] from the &#8942; button found on each virtual machine in the *Workloads* -> *Virtual Machines* list, or from the xref:vm-actions-web[*Actions*] button on the *Virtual Machine Details* screen.
```
Comment (fabiand) [inline web-interface/controlling-vms-modules/restart-vm-web.adoc]: Can we have an utf8 arrow instead of "->"?

**Comment by fabiand** [comment]
Thanks a lot!

@jelkosz @mareklibra @pcbailey who of you can tweet about this?

File: web-interface/controlling-vms-modules/restart-vm-web.adoc (hunk 1 above)
Comment (pcbailey) [reply web-interface/controlling-vms-modules/restart-vm-web.adoc]: It looks like '->' is automatically converted into a UTF-8 arrow when the docs are created. I'm happy to change it if you think something else would look better, though.

**Comment by pcbailey** [comment]
@fabiand I'll tweet about it once the fixes [1] I just submitted are merged.

[1] https://github.com/kubevirt/user-guide/pull/232

**Pull request merged**